### PR TITLE
Feature/pudding button

### PR DESCRIPTION
**Pull request by baspalinckx**
Added a button which will jiggle on hover, like a pudding!
Only issue is for the life of me, I can't get it to show up in sbutton.css. I do exactly as is described on the page. What am I missing? 

Issue: https://github.com/shahednasser/sbuttons/issues/498

### CODE REVIEW

File: src/components/animated/_pudding.less [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.pudding-btn() {
```
Comment (shahednasser) [inline src/components/animated/_pudding.less:1]: The problem is here. When you add `()` in less it creates a mixin not a class. Remove them to get it to work

**Review by shahednasser — CHANGES_REQUESTED** [review]
Make sure to compile after doing the change below

File: src/components/animated/_pudding.less (hunk 1 above)
Comment (baspalinckx) [reply src/components/animated/_pudding.less:1]: Of couse! How stupid of me

**Comment by baspalinckx** [comment]
Should be good now!

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -126,6 +126,11 @@ $(document).ready(function () {
           label: "Splash",
           classes: "splash-btn",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:129]: Looks cute but my only remark is to place this button in the correct alphabetical order under "Animated" (sorry I didn't notice this before).

Comment (baspalinckx) [reply assets/js/buttons-examples.js]: No problem. Should have spotted that

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @baspalinckx for idea

**Pull request merged**